src/js/actions/layers.js

```diff
@@ -134,6 +132,135 @@ define(function (require, exports) {
      */
     var METADATA_NAMESPACE = global.EXTENSION_DATA_NAMESPACE;
 
+    var multiGetRef = function (docRef, layerRef, properties) {
+        return {
+            name: "get",
+            descriptor: {
+                "null": {
+                    "_multiGetRef": [
+                        {
+                            _propertyList: properties
+                        },
+                        layerRef,
+                        docRef
+                    ]
+                }
+                
+            },
+            "options": {
+                useMultiGet: true,
+                failOnMissingProperty: false
+            }
+        };
+    };
```

Comment:
There should be a better way to do this instead of constructing the command manually.